armi/physics/neutronics/parameters.py

```diff
@@ -345,6 +317,38 @@ def mgFlux(self, value):
 
         pb.defParam("ppdensGamma", units="W/cm^3", description="Peak gamma density")
 
+    # rx rate params that are derived during mesh conversion.
+    # We'd like all things that can be derived from flux and XS to be
+    # in this category to minimize numerical diffusion but it is a WIP.
```

Comment:
not anymore because they're remade after mapping now, no longer being mapped. 

Maybe a better name for this category would be "remapDuringMeshConversion"